digdag-client/src/main/java/io/digdag/client/api/RestWorkflowDefinition.java

```diff
@@ -19,6 +21,9 @@
 
     String getRevision();
 
+    @JsonProperty("timezone")
+    ZoneId getTimeZone();
```

Comment:
I think using `getTimeZone` is OK for consistency with others: https://github.com/treasure-data/digdag/search?utf8=%E2%9C%93&q=getTimeZone